### PR TITLE
DatastoreIO statistics query fixes

### PR DESCRIPTION
**Pull request by vikkyrk**
- Back port from https://github.com/apache/incubator-beam/pull/868

### CODE REVIEW

**Comment by vikkyrk** [comment]
R: @dhalperi

File: sdk/src/test/java/com/google/cloud/dataflow/sdk/io/datastore/DatastoreV1Test.java [hunk 1]
```diff
@@ -191,14 +192,14 @@ public void testReadValidationFailsQueryLimitNegative() throws Exception {
...
-    DatastoreV1.Read read = DatastoreIO.v1().read().withProjectId(PROJECT_ID).withQuery(QUERY);
+    Read read =  DatastoreIO.v1().read().withProjectId(PROJECT_ID).withQuery(QUERY);
     /* Should succeed, as a null namespace is fine. */
     read.validate(null);
   }
 
   @Test
   public void testReadDisplayData() {
-    DatastoreV1.Read read = DatastoreIO.v1().read()
+    Read read =  DatastoreIO.v1().read()
```
Comment (dhalperi) [inline sdk/src/test/java/com/google/cloud/dataflow/sdk/io/datastore/DatastoreV1Test.java]: drop extra space?

**Comment by vikkyrk** [comment]
R: @dhalperi fixed.

**Comment by dhalperi** [comment]
LGTM , merged

**Pull request merged**